docs/hardware/srobo/pbv4.rst

```diff
@@ -160,7 +160,6 @@ use the board but it may be of interest to some people.
 .. _Hardware designs: https://www.studentrobotics.org/cgit/boards/power-v4-hw.git/
 
 
-.. Note:: Some of the above documentation has been taken and modified from the `Student Robotics`_ docs, which can be found here_.
+.. Note:: Some of the documentation in this section has been taken and modified from `Student Robotics`_. See here_ for more information.
 
-.. _Student Robotics: https://studentrobotics.org/
-.. _here: https://github.com/srobo/docs/blob/master/kit/power_board.md
\ No newline at end of file
+.. _here: LICENSE.html
```

Comment:
This link is different to the others, which just point to `LICENSE`. I'm guessing this one's correct and the other two are broken?

Reply:
You are right. Sorted, sorry for commit spam. It's on my list to learn how to do interactive rebasing properly.